addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -178,6 +187,9 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend(SurveyPre
         }
 
         this.currentScreen = screenToDisplay;
+        if (!['question', 'nextQuestion'].includes(screenToDisplay)) {
+            this._updateNextPageTooltip();
```

Comment:
Is it safe to do this, as we are not sure that "nextQuestion" is done, as it contains promises to go to the next question and we are not "awaiting" them ?

Or is that never happening as we exclude "question' and "nextQuestion" before?
In that case, how is the tooltip updated? By the refreshed question HTML? Or we don't want to update it at all?

So lots of questions, I think this deserves at least a code comment ^^

Reply:
Hello @awa

When we have the next question, the widget will be re-rendered.
so in that case we have flicker.
1) if we don't have this condition it will directly replace the tooltip without waiting for the question to be updated (so this won't look good)
2) from the start method when the question screen is changed.

Reply:
Then can you add a small comment as to why we do it?

Something like:

// To avoid a flicker, we do not update the tooltip when going to the next question, as it will be done in "_setupCurrentScreen"

Reply:
Hello @awa-odoo 

changes Done.

Thanks